vendor.mod

```diff
@@ -57,6 +57,7 @@ require (
 	github.com/imdario/mergo v0.3.13
 	github.com/ishidawataru/sctp v0.0.0-20230406120618-7ff4192f6ff2
 	github.com/klauspost/compress v1.16.3
+	github.com/mdlayher/packet v1.1.2
```

Comment:
I'm usually a fan of updating vendors separately (and it may be useful during review), but I guess this should be squashed with the commit that starts using it.

Technically this commit is incorrect, as running `hack/vendor.sh` on this commit will remove the files 😄